wrapanapi/systems/redfish.py

```diff
@@ -178,7 +311,57 @@ def get_server(self, resource_id):
         """
         return RedfishServer(self, raw=self.api_client.find(resource_id))
 
+    def get_chassis(self, resource_id, *required_types):
+        """
+        Fetch a RedfishChassis instance of the physical chassis representing resource_id.
+
+        In Redfish, a Chassis may be of specific type such as Rack, Sled, Block
+        or any other type. Use the required_types optional parameter to filter
+        by type, or use any of the specific getters such as get_rack.
+
+        Args:
+          resource_id: the Redfish @odata.id of the resource representing the
+             chassis to be retrieved
+          required_types: optional list of ChassisType values that the retrieved
```

Comment:
Its not clear to me here if these are just strings, or class constants?

Reply:
They are strings. I changed the description to hopefully make it clearer.